proposals/wac-ucr/index.bs

```diff
@@ -857,6 +857,18 @@ the following policy: their authors can edit comments for a minimum of 5
 minutes and only until these are themselves commented. After that point, the
 post will be in read-only access mode.
 
+### Conditional access by payment ### {#conditional-payment}
+
+A musician would like to self publish her music to take advantage of a
+Solid music App. She does this by making the song metadata available openly
+so that fans can follow her new releases, specialised music search engines
+can index it, and music apps can display the song's metadata in their
+users' preferred language. To earn a living, she sets an access control
+rule on the resource, allowing access on proof of payment. Payment methods
```

Comment:
Would that proof of payment include information which songs and/or albums were purchased?